README.md

```diff
@@ -137,8 +137,8 @@ Aleth supports EVMC out of the box, but geth not yet.
 	Here's the recommended configuration for connecting your node to the Ewasm testnet:
 
 	```sh
-	> EVMC_PATH=~/hera/build/src/libhera.so EVMC_OPTIONS='metering=true fallback=true' \
-	./build/bin/geth \
+	> ./build/bin/geth \
+	--vm.ewasm="/home/ewasm/hera/src/libhera.so:metering=true:fallback=true" \
```

Comment:
Actually the `libhera.so` path is wrong. It should be `hera/src/libhera`, but the build dir.

I think the original `~/hera/build/src/libhera.so` was slightly better.

Reply:
Even better: https://github.com/ewasm/testnet/pull/94#discussion_r228764637

Reply:
Use , not :

Reply:
fixed